starwars/src/CharList.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+
+export const CharList = ({ chars }) => {
+    return (
+      <div>
+        {chars.map((person => {
+          return (
+            <div key={person.count}>
+              <p>Name: {person.name}</p>
+              <p>Height: {person.height}</p>
+              <p>Hair Color: {person.hair_color}</p>
+              <p>Eye Color: {person.eye_color}</p>
+              <p>DOB: {person.birth_year}</p>
+              <p>gender: {person.gender}</p>
+              <p>Homeworld: {person.homeworld}</p>
+              <div>vehicles: {person.vehicles.map((film, i) => {
+                  console.log(film)
+                  return (
+                      <div key={i}>
+                        <p>{film}</p>
+                    </div>
+                  )
+              })}
+              </div>
+              <div>species: {person.species.map((species, i) => {
+                  console.log(species)
+                  return (
+                      <div key={i}>
+                        <p>{species}</p>
+                    </div>
+                  )
+              })}
+              </div>
+              <div>vehicles: {person.vehicles.map((vehicle, i) => {
```

Comment:
Why do you have `vehicles` twice? 🤔There's a `vehicles` already being mapped on line 16